### PR TITLE
Add support to bump A-C on focus-android release branches

### PR DESCRIPTION
**Pull request by jcristau**

### CODE REVIEW

**Review by escapewindow — APPROVED** [review]
Largely a stamp; let's monitor this after we roll out.
LMK if you want a deeper review?

File: src/fenix.py [hunk 1]
```diff
@@ -6,110 +6,22 @@
...
-    pr = fenix_repo.create_pull(
-        title=f"Update to Android-Components {latest_ac_version}.",
-        body=f"This (automated) patch updates Android-Components to {latest_ac_version}.",
-        head=pr_branch_name,
-        base=release_branch_name,
-    )
-    print(f"{ts()} Pull request at {pr.html_url}")
-
-
 def update_android_components(ac_repo, fenix_repo, author, debug, dry_run):
```
[the remark this answers is not in the document]
Comment (jcristau) [reply src/fenix.py:12]: nightly bumps in fenix are done via tc cron and https://github.com/mozilla-mobile/fenix/blob/main/automation/taskcluster/update_android_components.sh (which I don't love, but that's how it is for now).

File: src/fenix.py [hunk 2]
```diff
@@ -6,110 +6,22 @@
...
     for fenix_version in get_recent_fenix_versions(fenix_repo):
+        release_branch_name = f"releases_v{fenix_version}.0.0"
         try:
-            update_android_components_in_fenix(
-                ac_repo, fenix_repo, fenix_version, author, debug, dry_run
+            update_android_components_release(
```
[the remark this answers is not in the document]
Comment (jcristau) [reply src/fenix.py:16]: ack.  I'm refraining from massive refactors while we figure out what we want to do with this, just wanted a short-term fix for focus :)

**Comment by jcristau** [comment]
> Largely a stamp; let's monitor this after we roll out. LMK if you want a deeper review?

I think this is fine, thanks for the review.

**Pull request merged**